mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/148 - device property panel (merge this first)

Closes #148 

To test:
- Run introductory scenario
- From Network View click on BitFlipper Firewall
- Make sure properties display correctly
